index.js

```diff
@@ -55,11 +57,11 @@ const gatherInput = async () => {
 const writeCommit = async ({ message, coAuthors, otherArgs }) => {
   const coAuthoringLines = coAuthors.map(
     coAuthor =>
-      `-m "Co-authored-by: ${coAuthor} <${coAuthor.toLowerCase()}@users.noreply.github.com>"`
+      `Co-authored-by: ${coAuthor} <${coAuthor.toLowerCase()}@users.noreply.github.com>`
   )
-  const gitCommand = `git commit -m "${message}" ${coAuthoringLines.join(
-    ' '
-  )} ${otherArgs.join(' ')}`
+  const gitCommand = `git commit -m "${message}\n\n${coAuthoringLines.join(
```

Comment:
I’m not sure if those `\n` behave the same on all environments, but hey, it’d be a problem with the co authoring lines anyway 😄

Reply:
Pretty sure it's handled by Node.js, so should be fine :)

Reply:
I'll try this on Windows rig tonight though, just to be sure. :)

Reply:
I amma try https://coderwall.com/p/mrio6w/split-lines-cross-platform-in-node-js though, see if it works :)

Reply:
Ah yeah nice nice nice